src/main/java/frc/robot/OI.java

```diff
@@ -1,11 +1,51 @@
 package frc.robot;
 
+import edu.wpi.first.networktables.EntryListenerFlags;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+
+
 /**
  * This class is the glue that binds the controls on the physical operator
  * interface to the commands and command groups that allow control of the robot.
  */
 public class OI {
     public OI() {
+        createOperatorCommands();
+        NetworkTableEntry changeSettingsEntry = SmartDashboard.getEntry("isHillelSettings");
+        changeSettingsEntry.setBoolean(true);
+        // Listens for operator settings change. If it changes while the robot is disabled, the OI updates the button bindings.
+        changeSettingsEntry.addListener((entryNotification) -> {
+            if (DriverStation.getInstance().isDisabled()) {
+                CommandScheduler.getInstance().clearButtons();
+                if (entryNotification.value.getBoolean()) {
+                    setHillelSettings();
+                } else {
+                    setGrossmanSetting();
+                }
+            } else {
+                changeSettingsEntry.setBoolean(!entryNotification.value.getBoolean());
+            }
+        }, EntryListenerFlags.kNew | EntryListenerFlags.kImmediate | EntryListenerFlags.kUpdate);
+    }
+
+    private void createOperatorCommands() {
```

Comment:
private void createOperatorCommands() {
me and Grossman are the Operators... Yoav is the Driver:
    private void createDriverCommands() {

Reply:
But Yoav commands should be switched. So we can just use buttons like this:
``` java
public class OI {
    private TrigonXboxController driverXbox;
    public OI() {
        createOperatorCommands();
        driverXbox = new TrigonXboxController(0);
        driverXbox.getButtonA().whenPressed(new InstantCommand());
        ...
    }
}
```
Instead of:
``` java
public class OI {
    private TrigonXboxController driverXbox;
    private InstantCommand instantCommmand
    public OI() {
        createOperatorCommands();
        driverXbox = new TrigonXboxController(0);
        driverXbox.getButtonA().whenPressed(instantCommand);
        ...
    }
    private void createDriverCommands() {
        instantCommand = new InstantCommand();
    }
}
```
It just makes the code longer

Reply:
The point is that we want to create commands once and bind them to buttons when the button is pressed. I don't think it's a good idea to create new commands when we change the button bindings. In order to create all of the commands only one time, we need to have a method that creates it and saves it to the class fields.

Reply:
oh, yes I agree with that I just wanted to change the name